docs/whats-new.rst

```diff
@@ -1,6 +1,12 @@
 What's New
 ==========
 
+v0.1.1
+------
+- Bug fixes related to the start of the ``calc.py`` refactor in :issue:`90`
+  (see :issue:`126` and :issue:`128`)
+- Compatability with xarray version 0.9.0
+
```

Comment:
No problem, I updated things to be more detailed.  Let me know if that looks good!